spec/controllers/api/v3/analytics/user_analytics_controller_spec.rb

```diff
@@ -63,30 +62,30 @@
 
         context 'achievements' do
           it 'has the section visible' do
-            Timecop.freeze("10:00 AM UTC") do
-              #
-              # create BPs (follow-ups)
-              #
-              patients = create_list(:patient, 3, registration_facility: request_facility)
-              patients.each do |patient|
-                [patient.recorded_at + 1.month,
-                patient.recorded_at + 2.months,
-                patient.recorded_at + 3.months,
-                patient.recorded_at + 4.months].each do |date|
-                  travel_to(date) do
-                    create(:encounter,
-                          :with_observables,
-                          observable: create(:blood_pressure,
-                                              patient: patient,
-                                              facility: request_facility,
-                                              user: request_user))
-                  end
+            request_date = Date.new(2018, 4, 8)
```

Comment:
This is similar to to @rsanheim's fix from before, instead it fixes a date in the middle of the month instead of a time in the middle of the day.

Reply:
The reason why we need this here, but not in a very similar spec in the [presenter](https://github.com/simpledotorg/simple-server/pull/920/files#diff-48e7d5ca923c09ab96f49ea4449f4ba1R292) is because the presenter does not go through the controller's tz-binding code.